pkg/trace/api/normalizer_test.go

```diff
@@ -322,10 +374,10 @@ func TestNormalizeInvalidUTF8(t *testing.T) {
 	t.Run("service", func(t *testing.T) {
 		assert := assert.New(t)
 
-		span := testSpan()
+		ts, span := &info.TagStats{}, newTestSpan()
```

Comment:
Can we please not do this and instead just:
```go
ts := &info.TagStats()
span := newTestSpan()
```

Reply:
Sure, updated